core/lib/Thelia/Action/Product.php

```diff
@@ -100,6 +121,451 @@ public function create(ProductCreateEvent $event)
         $event->setProduct($product);
     }
 
+    /*******************
+     * CLONING PROCESS *
+     *******************
+     *
+     * Vars prefixes :
+     * op = Original Product
+     * cp = Clone Product
```

Comment:
Use full variable name please, It's easier to read and understand the code.

Thanks